### PR TITLE
location and county model tests

### PR DESCRIPTION
**Pull request by dsai96**
One test is failing, but will look into it

### CODE REVIEW

**Comment by dsai96** [comment]
The test failing is a schema issue that @glaser06 said he will address. so for now I have commented it out.

**Comment by RebeccaKern** [comment]
I'm going to start reviewing now, but you'll also have to resolve the routes conflicts

File: app/models/location.rb [hunk 1]
```diff
@@ -11,6 +11,8 @@ class Location < ApplicationRecord
   accepts_nested_attributes_for :addresses, reject_if: lambda { |addr| addr[:address].blank? }, allow_destroy: true
 
   validates :name, presence: true
```
Comment (RebeccaKern) [inline app/models/location.rb:12]: Location needs to validate more than just name

File: test/test_helper.rb [hunk 2]
```diff
@@ -1,10 +1,35 @@
...
-
+  # fixtures :all
   # Add more helper methods to be used by all tests here...
+
+  def deny(condition)
+    # a simple transformation to increase readability IMO
+    assert !condition
+  end
+
+  def create_locations
```
Comment (RebeccaKern) [inline test/test_helper.rb:20]: Shouldn't we test that certain things don't build?

File: app/models/location.rb (hunk 1 above)
Comment (dsai96) [reply app/models/location.rb:12]: Yeah i know, I wasn't done completely with location tests, but just wanted to check there were no merge conflicts with what you guys had done this week so i just decided to push

File: test/test_helper.rb (hunk 2 above)
Comment (dsai96) [reply test/test_helper.rb:20]: i don't think that's necessary really, but we could add them in if you think so

**Pull request merged**